techniques/aria/ARIA24.html

```diff
@@ -289,25 +272,24 @@ <h4>User CSS</h4>
                   	</section>
                <section id="working-example">
                   
-                  <h4>Working example:</h4>
+                  <h4>実例:</h4>
                   
                   <p>
-                     <a href="../../working-examples/aria-icon-font-img-role/#example3">Email Icon Font in a link WITHOUT visible text</a>
+                     <a href="https://www.w3.org/WAI/WCAG21/working-examples/aria-icon-font-img-role/#example3">表示テキストのない、リンク付きメールのアイコンフォント</a>
                      
                   </p>
                   	
                </section>
             </section>
             <section class="example" id="example-4-multiple-icon-fonts-as-part-of-another-sematic-element-with-visible-text">
-               <h3>Example 4: Multiple Icon Fonts as part of another sematic element WITH visible text</h3>
-               <p>This example already has a visible text label in the link to be used as an accessible
-                  name, the mail and chevron font icons must stay visible when the font family is changed.
-                  This can be done by ensuring the icons are contained in their own element and the
-                  attribute <code>aria-hidden="true"</code> is used so the font icons will be ignored by assistive technologies.
+               <h3>実例 4: 表示テキストのあるセマンティック要素の一部として、複数のアイコンフォント</h3>
+               <p>この例では、アクセシブルな名前 (accessible name)として使用するために、リンクにテキストラベルが既に表示されている。
+                  フォントファミリーが変更されても、メールとシェブロンのフォントアイコンは表示されていなければならない。
+                  これは、アイコンがそれぞれの要素に含まれ、又、支援技術によってフォントアイコンが無視されるように属性 <code>aria-hidden="true"</code>  が使用されることで実現できる。
```

Comment:
```suggestion
                  これは、アイコンがそれぞれの要素に含まれ、かつ、支援技術によってフォントアイコンが無視されるように属性 <code>aria-hidden="true"</code>  が使用されることで実現できる。
```

日本語訳で改行があるので改行を取り除いてください。